user-mgt/custom-jdbc-user-store-manager/pom.xml

```diff
@@ -89,6 +90,11 @@
             <artifactId>org.wso2.carbon.user.core</artifactId>
             <version>${carbon.kernel.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.jasypt</groupId>
+            <artifactId>jasypt</artifactId>
+            <version>1.9.2</version>

```

Comment:
Add version tag at the end of the pom file. Similar to ` <version>${carbon.kernel.version}</version>`

Reply:
Fixed with https://github.com/wso2/samples-is/pull/330/commits/d9dc0341d2536a042f541d3a1afab4584d3a96eb